src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -45,7 +45,10 @@ class PopulationNearRoadsJob(
   @transient lazy val layoutTileSource: LayoutTileSource[SpatialKey] =
     LayoutTileSource.spatial(rasterSource, layout)
 
-  val wsCountryBorder: MultiPolygon = country.boundary.reproject(LatLng, layoutTileSource.source.crs)
+  val wsCountryBorder: Geometry = {
+    // Buffer country border to avoid clipping out tiles where WorldPop and NaturalEarth don't agree on borders
+    country.boundary.buffer(2).reproject(LatLng, layoutTileSource.source.crs)
```

Comment:
If we're going to merge this and knowingly introduce an issue where we can no longer run all countries in one pass, we should note that in the README or as an issue so we don't lose sight of it for later.